### DISCUSSION TITLE
add image size edit in the rich text editor

### DISCUSSION THREAD

**Discussion by chenshangwei** [directus/directus#18695] [Feature Requests]
### Summary

until now，Image sizes cannot be edited in the rich text editor

### Motivation

Itis Good for text editing

### Detailed Design

Select an image in the editor and zoom in or out

### Requirements List

#

### Drawbacks

#

### Alternatives

#

### Adoption Strategy

#

**Comment by rijkvanzanten** [comment]
Closing as there's not enough information to discuss
